packages/eslint-plugin-template/src/rules/prefer-ngsrc.ts

```diff
@@ -0,0 +1,68 @@
+import type { TmplAstElement } from '@angular-eslint/bundled-angular-compiler';
+import { getTemplateParserServices } from '@angular-eslint/utils';
+import type {
+  TmplAstBoundAttribute,
+  TmplAstTextAttribute,
+} from '@angular/compiler';
+import { createESLintRule } from '../utils/create-eslint-rule';
+
+type Options = [];
+export type MessageIds = 'missingAttribute' | 'invalidDoubleSource';
+export const RULE_NAME = 'prefer-ngsrc';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Ensures ngSrc is used instead of src for img elements',
+      recommended: false,
+    },
+    schema: [],
+    messages: {
+      missingAttribute:
+        'The attribute [ngSrc] should be used for img elements instead of [src].',
+      invalidDoubleSource:
+        'Only [ngSrc] should exist on an img element. Delete the [src] attribute.',
+    },
+  },
+  defaultOptions: [],
+  create(context) {
+    const parserServices = getTemplateParserServices(context);
+
+    return {
+      'Element$1[name=img]'(element: TmplAstElement) {
+        const ngSrcAttribute = hasNgSrcAttribute(element);
+        const srcAttribute = hasNormalSrcAttribute(element);
+
+        if (srcAttribute == null) {
+          return;
+        }
+
+        const loc = parserServices.convertNodeSourceSpanToLoc(
+          srcAttribute.sourceSpan,
+        );
+
+        context.report({
+          loc,
+          messageId:
+            ngSrcAttribute == null ? 'missingAttribute' : 'invalidDoubleSource',
```

Comment:
Whilst functional, this is an unintuitive choice of condition for checking a value which will never be `null`.

Let's prefer simple code:

```suggestion
            !ngSrcAttribute ? 'missingAttribute' : 'invalidDoubleSource',
```